### PR TITLE
Design for webhook administration (rebased)

### PR DESCRIPTION
**Pull request by znarf**
Rebasing and editing previous PR:
https://github.com/opencollective/opencollective-frontend/pull/2103

### CODE REVIEW

File: src/components/EditWebhooks.js [hunk 1]
```diff
@@ -192,44 +255,66 @@ class EditWebhooks extends React.Component {
...
+            buttonStyle="primary"
+            buttonSize="medium"
             onClick={this.handleSubmit}
             disabled={loading || !this.state.modified}
+            loading={status == 'loading'}
+            px={4}
           >
-            {submitBtnLabel}
-          </Button>
+            {captializedSubmitBtnLabel}
```
Comment (Betree) [inline src/components/EditWebhooks.js]: As per https://docs.opencollective.com/help/developers/translations#dont-assume-words-order-stays-the-same-in-other-languages we shouldn't assume word's order stays the same in other languages.

We need to have a dedicated string here, something like:

```es6
<FormattedMessage
  id="webhooks.Save"
  defaultMessage="Save {count} webhooks"
  values={{count: webHookCount}}
/>
```

Comment (NotJohs) [reply src/components/EditWebhooks.js]: @Betree  thanks a lot, can i still work on this issue

> **Code**
> 
> * Should use `styled-components`
> 
> **UI/UX**
> 
> * Webhooks aren't sent to API when saving
> * No button to remove webhook when there's only 1
> * Responsiveness issue
> 
> image

**Comment by NotJohs** [comment]
@Betree thanks a lot. Understood.

File: src/components/EditWebhooks.js [hunk 2]
```diff
@@ -192,44 +255,66 @@ class EditWebhooks extends React.Component {
 
     let submitBtnMessageId = 'save';
     if (['loading', 'saved'].includes(status)) {
-      submitBtnMessageId = status;
+      submitBtnMessageId = 'saved';
     }
     const submitBtnLabel = this.messages[submitBtnMessageId] && intl.formatMessage(this.messages[submitBtnMessageId]);
+    const submitBtnLabelEnd = this.messages['title'] && intl.formatMessage(this.messages['title']);
```
[the remark this answers is not in the document]
Comment (NotJohs) [reply src/components/EditWebhooks.js]: A mistake. will Fix.

**Comment by NotJohs** [comment]
@znarf @Betree sorry for the delay. Learning the styled-system and implementing  it is a bit tricky but i am making progress.

**Pull request closed without merging**